manifests/rbac/rbac-namespaced.yaml

```diff
@@ -0,0 +1,20 @@
+# This manifest is auto-generated from the helm chart, do not modify! 
+# Source: amalthea/templates/rbac/rbac-namespaced.yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: amalthea
+  namespace: default
+  labels:
+    helm.sh/chart: amalthea-0.0.1-set.by.chartpress
```

Comment:
@ableuler I think this may be another thing that should be purged from the manifests?

Reply:
This shows up in almost all rendered manifests.